components/ide/jetbrains/backend-plugin/src/main/resources/META-INF/plugin.xml

```diff
@@ -22,18 +22,114 @@
+                     restartRequired="true"/>
     </extensions>
 
+    <actions>
+        <action id="io.gitpod.jetbrains.remote.actions.DashboardAction"
+                class="io.gitpod.jetbrains.remote.actions.DashboardAction"
+                text="Gitpod: Open Dashboard"
+                icon="AllIcons.General.ProjectTab">
+            <add-to-group group-id="UnattendedHostDropdownGroup" anchor="last"/>
+        </action>
+        <action id="io.gitpod.jetbrains.remote.actions.ContextAction"
+                class="io.gitpod.jetbrains.remote.actions.ContextAction"
+                text="Gitpod: Open Context"
+                icon="AllIcons.General.FitContent">
+            <add-to-group group-id="UnattendedHostDropdownGroup" anchor="last"/>
+        </action>
+        <action id="io.gitpod.jetbrains.remote.actions.SettingsAction"
+                class="io.gitpod.jetbrains.remote.actions.SettingsAction"
+                text="Gitpod: Open Settings"
+                icon="AllIcons.General.Settings">
+            <add-to-group group-id="UnattendedHostDropdownGroup" anchor="last"/>
+        </action>
+        <action id="io.gitpod.jetbrains.remote.actions.AccessControlAction"
+                class="io.gitpod.jetbrains.remote.actions.AccessControlAction"
+                text="Gitpod: Open Access Control"
+                icon="AllIcons.Actions.Properties">
+            <add-to-group group-id="UnattendedHostDropdownGroup" anchor="last"/>
+        </action>
+        <action id="io.gitpod.jetbrains.remote.actions.DocumentationAction"
+                class="io.gitpod.jetbrains.remote.actions.DocumentationAction"
+                text="Gitpod: Documentation"
+                icon="AllIcons.General.ReaderMode">
+            <add-to-group group-id="UnattendedHostDropdownGroup" anchor="last"/>
+        </action>
+        <action id="io.gitpod.jetbrains.remote.actions.ReportIssueAction"
+                class="io.gitpod.jetbrains.remote.actions.ReportIssueAction"
+                text="Gitpod: Report Issue"
+                icon="AllIcons.General.ShowWarning">
+            <add-to-group group-id="UnattendedHostDropdownGroup" anchor="last"/>
+        </action>
+        <action id="io.gitpod.jetbrains.remote.actions.FollowUsOnTwitterAction"
+                class="io.gitpod.jetbrains.remote.actions.FollowUsOnTwitterAction"
+                text="Gitpod: Follow Us On Twitter"
+                icon="AllIcons.Gutter.ExtAnnotation">
+            <add-to-group group-id="UnattendedHostDropdownGroup" anchor="last"/>
+        </action>
+        <action id="io.gitpod.jetbrains.remote.actions.CommunityChatAction"
+                class="io.gitpod.jetbrains.remote.actions.CommunityChatAction"
+                text="Gitpod: Open Community Chat"
+                icon="AllIcons.General.Balloon">
+            <add-to-group group-id="UnattendedHostDropdownGroup" anchor="last"/>
+        </action>
+        <action id="io.gitpod.jetbrains.remote.actions.InBrowser"
```

Comment:
The "Open in Browser" is a weird one, because it actually opend supervisor-frontend which then triggers again the opening of JetBrains itself.

It would be cool if we could offer "Open in VS Code Browser" 👀 

Otherwise, I think it's better to remove it, unless I am missing something? cc @akosyakov

Reply:
I agree. Also felt it was not fitting in JetBrains. And couldn't find a way to directly open VS Code when clicking the action. So I'm going to comment it out, and it can be removed in the follow-up PR.

Reply:
I don't think we should have any mentions of VS Code Browser in IntelliJ. `Open in Browser` sounds good. We also can allow other IDEs in the future. I think for start we can also just drop it.

Reply:
Opening the browser will trigger automatically the JB to open the thinclient, I don't think this action is useful at all, unless I am misunderstanding what it should do? cc @akosyakov

Reply:
let's remove, if we do it indeed we need some indicators to prevent auto openning.